cinema/views.py

```diff
@@ -1,45 +1,154 @@
+            pk=pk)
+        return genre
 
-@api_view(["GET", "PUT", "DELETE"])
-def movie_detail(request, pk):
-    movie = get_object_or_404(Movie, pk=pk)
+    def get(self, request, pk):
+        genre = self.get_object(pk)
+        serializer = GenreSerializer(genre)
+        return Response(
+            serializer.data,
+            status=status.HTTP_200_OK)
 
-    if request.method == "GET":
-        serializer = MovieSerializer(movie)
-        return Response(serializer.data, status=status.HTTP_200_OK)
+    def put(self, request, pk):
+        genre = self.get_object(pk)
+        serializer = GenreSerializer(genre, data=request.data)
 
-    if request.method == "PUT":
-        serializer = MovieSerializer(movie, data=request.data)
         if serializer.is_valid():
             serializer.save()
-            return Response(serializer.data, status=status.HTTP_200_OK)
+            return Response(
+                serializer.data,
+                status=status.HTTP_200_OK)
 
-        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+    def patch(self, request, pk):
+        genre = self.get_object(pk)
+        serializer = GenreSerializer(genre, data=request.data, partial=True)
 
-    if request.method == "DELETE":
-        movie.delete()
+        if serializer.is_valid():
+            serializer.save()
+            return Response(
+                serializer.data,
+                status=status.HTTP_200_OK)
+
+    def delete(self, request, pk):
+        genre = self.get_object(pk)
+        genre.delete()
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class ActorList(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    generics.GenericAPIView
+):
+    queryset = Actor.objects.all()
+    serializer_class = ActorSerializer
+
+    def get(
+            self, request,
+            *args, **kwargs):
+        return self.list(
+            request, *args,
+            **kwargs)
+
+    def post(
+            self, request,
+            *args, **kwargs):
+        return self.create(
+            request, *args,
+            **kwargs)
+
+
+class ActorDetail(
+    mixins.RetrieveModelMixin,
+    mixins.UpdateModelMixin,
+    mixins.DestroyModelMixin,
+    generics.GenericAPIView
+):
+    queryset = Actor.objects.all()
+    serializer_class = ActorSerializer
+
+    def get(
+            self, request,
+            *args, **kwargs):
+        return self.retrieve(
+            request, *args,
+            **kwargs)
+
+    def put(
+            self, request,
+            *args, **kwargs):
+        return self.update(
+            request, *args,
+            **kwargs)
+
+    def patch(
+            self, request,
+            *args, **kwargs):
+        return self.partial_update(
+            request, *args,
+            **kwargs)
+
+    def delete(
+            self, request,
+            *args, **kwargs):
+        return self.destroy(
+            request, *args,
+            **kwargs)
```

Comment:
why u start it all from new line? what line length do you have in your flake file or IDE setiings?